README.md

```diff
@@ -65,16 +66,21 @@ To this end, this provider supports the following extra specs schema:
         },
         "enable_boot_debug": {
             "type": "boolean",
-            "description": "Enable cloud-init debug mode. Adds `set -x` into the cloud-init script."
+            "description": "Enable cloud-init debug mode. Adds 'set -x' into the cloud-init script."
         },
-        "allow_image_owners": {
+        "allowed_image_owners": {
```

Comment:
I get why this change is made, but keep in mind that people are already using the old naming. Changing this will break their pools when they update.

Reply:
In the code itself it's `allowed_image_owners`, but in the Readme was `allow_image_owners`. Which one should we stick to then?

Reply:
ohh! The one in the code. Aesthetics matter less. We don't want to change functionality for the sake of aesthetics, but if the code is okay and the README has a typo, ignore me.